tools/Dockerfile

```diff
@@ -0,0 +1,110 @@
+#
```

Comment:
Need to delete this, or rather, we need to decide where the Dockerfile should live.